benchmark/benchmarks.jl

```diff
@@ -0,0 +1,12 @@
+# Each file of the form "bench_$(name).jl" in this directory is `include`d and
+# its last statement is assumed to be a `BenchmarkGroup`.  This group is added
+# to the top-level group `SUITE` with the `$name` extracted from the file name.
+
+using BenchmarkTools
+SUITE = BenchmarkGroup()
```

Comment:
could be `const` I guess?

Reply:
OK, initially I didn't want `const` to avoid seeing the redefinition warning but it's not a good motivation anymore since I use modules.